enterprise/actions.py

```diff
@@ -58,3 +58,18 @@ def export_as_csv(modeladmin, request, queryset):  # pylint: disable=unused-argu
 
     export_as_csv.short_description = description
     return export_as_csv
+
+
+def get_clear_catalog_id_action(description=None):
```

Comment:
Instead of saying "Return a method..." in the docstring, we could add a little more detail to clear up any confusion -- something as simple as "Return the action method..." would probably be fine.